Signal/src/ViewControllers/ThreadSettings/OWSConversationSettingsViewController.m

```diff
@@ -247,6 +250,11 @@ - (void)viewDidLoad
             [[OWSDisappearingMessagesConfiguration alloc] initDefaultWithThreadId:self.thread.uniqueId];
     }
 
+    NSString *colorName = self.thread.conversationColorName;
+    OWSConversationColor *currentConversationColor = [OWSConversationColor conversationColorOrDefaultForColorName:colorName];
+    self.colorPicker = [[OWSColorPicker alloc] initWithCurrentConversationColor:currentConversationColor];
```

Comment:
Please make sure that the test plan includes entering/exiting the picker multiple times since we don't usually re-use views like this.